ice.c

```diff
@@ -2418,11 +2418,14 @@ void janus_ice_candidates_to_sdp(janus_ice_handle *handle, char *sdp, guint stre
 #endif
 			}
 		}
-		g_strlcat(sdp, buffer, JANUS_BUFSIZE);
+#pragma GCC diagnostic ignored "-Wformat-security"
```

Comment:
Good point and good suggestion! Will fix this in `sdp.c` too, where we have others.